store/server/features/timesheet/dashboard/queries.ts

```
@@ -0,0 +1,192 @@
+import { useQuery } from 'react-query';
+
+import { crudRequest } from '@/utils/crudRequest';
+import { TIME_AND_ATTENDANCE_URL } from '@/utils/constants';
+import { requestHeader } from '@/helpers/requestHeader';
+import { DASHBOARD_API } from './api';
+
+export const getAttendanceStats = async (userId: string) => {
+    return await crudRequest({
```

ESLint

[error] Delete `··`

(prettier/prettier)